examples/quickstart-tensorflow/README.md

```diff
@@ -15,39 +15,28 @@ This will create a new directory called `quickstart-tensorflow` containing the f
 
 ```shell
 -- pyproject.toml
--- requirements.txt
 -- client.py
 -- server.py
 -- README.md
 ```
 
 ### Installing Dependencies
 
-Project dependencies (such as `tensorflow` and `flwr`) are defined in `pyproject.toml` and `requirements.txt`. We recommend [Poetry](https://python-poetry.org/docs/) to install those dependencies and manage your virtual environment ([Poetry installation](https://python-poetry.org/docs/#installation)) or [pip](https://pip.pypa.io/en/latest/development/), but feel free to use a different way of installing dependencies and managing virtual environments if you have other preferences.
-
-#### Poetry
+Project dependencies (such as `tensorflow` and `flwr`) are defined in `pyproject.toml`. You can install the dependencies by invoking `pip`:
 
 ```shell
-poetry install
-poetry shell
+# From a new python environment, run:
+pip install .
 ```
 
-Poetry will install all your dependencies in a newly created virtual environment. To verify that everything works correctly you can run the following command:
+Then, to verify that everything works correctly you can run the following command:
 
 ```shell
-poetry run python3 -c "import flwr"
+python -c "import flwr"
```

Comment:
@danieljanes @jafermarq Have we taken a decision for replacing `python3` with `python`?

Reply:
I think the last time we discussed you recommended using `python3`? I always use `python` though

Reply:
I would assume that the majority of users would have `python` aliased to `python3` by default. I'm in favour of just specifying `python` for simplicity since it's one character shorter 😄 .

Reply:
Updated to `python3`.